contracts/token/ERC721/ERC721.sol

```diff
@@ -195,10 +195,11 @@ abstract contract ERC721 is Context, ERC165, IERC721, IERC721Metadata, IERC721Er
 
     /**
      * @dev Checks if `spender` can operate on `tokenId`, assuming the provided `owner` is the actual owner.
-     * Reverts if `spender` has not approval for all assets of the provided `owner` nor the actual owner approved the `spender` for the specific `tokenId`.
+     * Reverts if `spender` has not approval for all assets of the provided `owner` nor the actual owner approved
```

Comment:
“has not approval” -> “does not have approval”?

Reply:
```suggestion
     * Reverts if `spender` does not have approval from the provided `owner` for the given token or for all its assets
```
?